node-watcher/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "exclude": ["**/generated/**/*", "node_modules", "public", ".cache"],
+  "include": ["./src"],
+  "compilerOptions": {
+    "baseUrl": ".",
+    "declaration": true,
+    "downlevelIteration": true,
+    "esModuleInterop": true,
+    "jsx": "react",
+    "lib": ["dom", "esnext"],
```

Comment:
not now as it might affect current running setup but we should change tsconfig to be same as other backend services like chain db watcher. 

dom lib is not needed for backend

Reply:
right, tsconfig, eslint etc should be adapted obviously.